tests/cli/RunCest.php

```diff
@@ -190,6 +190,57 @@ public function runTestWithDataProviders(\CliGuy $I)
         $I->seeInShellOutput("OK");
     }
 
+    public function runTestWithDataProvidersFailure(\CliGuy $I)
+    {
+        $I->executeCommand('run tests/dummy/DataProvidersFailureTest.php --no-exit -v');
+        $I->seeInShellOutput('There was 1 warning');
+        $I->seeInShellOutput('PHPUnit_Framework_Warning');
+        $I->seeInShellOutput('The data provider specified for DataProvidersFailureTest::testIsTriangle is invalid.');
+        $I->seeInShellOutput('Method rectangle does not exist');
+        $I->seeInShellOutput("WARNINGS!");
+        $I->seeInShellOutput("Tests: 1, Assertions: 0, Warnings: 1.");
+    }
+
+    public function runTestWithDataProvidersExceptionStdout(\CliGuy $I)
+    {
+        $I->executeCommand('run tests/dummy/DataProvidersExceptionCest.php 2> /dev/null', false);
+        $I->dontSeeInShellOutput('There was 1 warning');
+        $I->dontSeeInShellOutput('PHPUnit_Framework_Warning');
+        $I->dontSeeInShellOutput('The data provider specified for DataProvidersExceptionTest::testIsTriangle is invalid.');
+        $I->dontSeeInShellOutput("WARNINGS!");
+        $I->dontSeeInShellOutput("OK");
+        $I->dontSeeInShellOutput("[Exception]");
+        $I->dontSeeInShellOutput("Something went wrong!!!");
```

Comment:
This test deliberately redirects StdErr which is where all the good stuff appears, the StdOut only looks like this:
```
$ codecept run tests/failure_exceptions/DataProvidersExceptionCest.php 2> /dev/null
Codeception PHP Testing Framework v2.3.4
Powered by PHPUnit 5.7.21 by Sebastian Bergmann and contributors.

$
```
I shied away from testing for the header/versions as they are likely to change. Could add a test for "Codeception" and "Powered by PHPUnit"?